Fibonacci/task.md

```diff
@@ -1,7 +1,20 @@
-Write a class called `Fibonacci` within the methode `calcFib (int n)` which returns
+Write a class called `Fibonacci` within the function `fibonacci (int n)` which returns
 the result of the Fibonacci function of given number `n`.
 
-Make sure that only legal arguments will be accepted otherwise throw an Exception `Fibonacci is not defined for neagtive Numbers!`.
+Make sure that only legal arguments will be accepted otherwise throw an
+IllegalArgumentException `Fibonacci is not defined for negative Numbers!`.
+
+## Example ##
+
+Here are some values to show, which results the function `fibonacci (n)` should return.
+
+`n` --> `fibonacci (n)`  
```

Comment:
Good idea! But why don't you use a Markdown table (see https://guides.github.com/features/mastering-markdown/)?